src/lib/builder.ts

```diff
@@ -131,6 +153,7 @@ export async function build() {
     build: {
       ssr: true,
       rollupOptions: {
+        onwarn,
```

Comment:
As I understand, this will overwrite previously defined option. I'm not sure if Vite provides a way to only merge the option if unavailable. Let me know if there's such a way.